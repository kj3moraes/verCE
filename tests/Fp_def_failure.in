def(x) e - x + 1
