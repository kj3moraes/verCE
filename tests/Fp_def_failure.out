DEF:def 
LPAREN:( 
ID:x 
RPAREN:) 
ID:e 
MINUS:- 
ID:x 
PLUS:+ 
NUM:1 
//:eof://:eof 
Handling definition

Compilation failed!
ERROR: Expected function name in prototype
